package.json

```diff
@@ -133,34 +127,29 @@
     "web components"
   ],
   "scripts": {
-    "build": "npm-run-all build:sass sass:render dist:js build:api bundler postinstall test",
+    "build": "npm-run-all build:sass sass:render dist:js build:api bundler postinstall test types",
     "build:ci": "npm-run-all sweep build",
     "build:api": "wca analyze 'src/auro-avatar.js' --outFiles docs/api.md",
-    "build:demo": "npm-run-all build demo:rm:build demo:new:build demo:copy:index demo:copy:demo demo:update:index",
     "build:dev:assets": "npm-run-all build:sass:component postCss:component sass:render",
     "build:sass": "npm-run-all build:sass:component postCss:component sass:render",
     "build:sass:component": "sass --no-source-map src:src",
     "build:watch": "nodemon -e scss,js --watch src --exec npm run build:dev:assets",
     "bundler": "rollup -c",
     "bundler:test": "rollup -c -w",
     "scssLint": "stylelint \"./src/**/*.scss\"",
-    "demo:copy:demo": "copyfiles -u 1 -V './demo/demo.md' ./build/demo",
-    "demo:copy:index": "copyfiles -u 1 -V './demo/index.html' ./build",
-    "demo:new:build": "mkdir ./build && mkdir ./build/css && mkdir ./build/demo",
-    "demo:rm:build": "rm -rf ./build",
-    "demo:update:index": "node ./scripts/prepForBuild",
-    "dev": "npm run build:watch",
+    "dev": "concurrently --kill-others 'npm run build:watch' 'npm run serve'",
```

Comment:
Last changes I forgot to make the run dev in a single terminal only, so I have added it now.